WordPress/Classes/ViewRelated/Support/SupportCoordinator.swift

```diff
@@ -0,0 +1,67 @@
+import Foundation
+import WordPressAuthenticator
+
+final class SupportCoordinator {
+    private weak var controllerToShowFrom: UIViewController?
+    private var tag: WordPressSupportSourceTag?
+
+    private var navigationController: UINavigationController? {
+        guard let navigationController = (controllerToShowFrom as? UINavigationController) ?? controllerToShowFrom?.navigationController else {
+            return nil
+        }
+
+        return navigationController
+    }
+
+    init(controllerToShowFrom: UIViewController?,
+         tag: WordPressSupportSourceTag? = nil) {
+        self.controllerToShowFrom = controllerToShowFrom
+        self.tag = tag
+    }
+
+    func showSupport(onIdentityUpdated: (() -> ())? = nil) {
+        guard let controllerToShowFrom = controllerToShowFrom else { return }
+
+        ZendeskUtils.sharedInstance.setPresentingViewController(controllerToShowFrom)
+
+        if RemoteFeatureFlag.contactSupportChatbot.enabled() {
```

Comment:
```suggestion
        if AppConfiguration.isJetpack && RemoteFeatureFlag.contactSupportChatbot.enabled() {
```

Without this, the chatbot is also shown on the WordPress app. I don't think we can use the bot since it's only trained to provide Jetpack app support.

The scenario where the user tries to close their account but has active purchases is the only place I'm aware of where the WordPress app still provides Zendesk support (instead of the forum).

Reply:
> The scenario where the user tries to close their account but has active purchases is the only place I'm aware of where the WordPress app still provides Zendesk support (instead of the forum).

I just noticed that WPAndroid sends users to the forum instead (tested on a build from https://github.com/wordpress-mobile/WordPress-Android/pull/19000).

Reply:
Throughout the JP and WP apps, the user is taken to the Help screen (instead of directly to the Zendesk screen or the forum) whenever they contact support. I remember before the forum, we'd open the Zendesk screen directly, but when the forums were made we changed this to keep the forum vs. Zendesk logic on one screen. The exceptions to this are `AccountSettingsViewController` and `GutenbergViewController` which go straight to the

If we align with Android, I think we bring both these exceptions in line with the rest of the app. It also aligns

Reply:
Okay. If I understand correctly, the suggestion is in all the contexts open Help & Support view when users tap "Contact Support".

> The exceptions to this are AccountSettingsViewController and GutenbergViewController.

I don't know why `GutenbergViewController` was an exception but `AccountSettingsViewController` probably still needs to be an exception, given that the account can be only closed via support. 🤔

> If we align with Android, I think we bring both these exceptions in line with the rest of the app. It also aligns

@ravishanker @irfano What is your view on this? Should we be always opening Help & Support view instead of directly opening chat/forum? What about closing the account case? Should it be the only exception where we always allow to contact Zendesk no matter the app or situation?

Reply:
Okay, so `GutenbergViewController` has 2 buttons:
- Contact Support
- More contact options

That's why it has both direct entry to Zendesk and can open Help & Support view.

The only thing then that we need to agree on is how we deal with `AccountSettings`.

Reply:
> but `AccountSettingsViewController` probably still needs to be an exception, given that the account can be only closed via support. 🤔

Good point! Sounds like we'll maintain this exception (ref p1692341248436899/1692309258.915019-slack-C05ALSVN5K5) but that's out of scope here. 

I'll mark this thread resolved now since we're now limiting the chatbot to the JP app.